internal/controller/csiaddons/persistentvolumeclaim_controller.go

```diff
@@ -114,6 +118,11 @@ func (r *PersistentVolumeClaimReconciler) Reconcile(ctx context.Context, req ctr
 		return ctrl.Result{}, nil
 	}
 
+	err = r.processKeyRotation(ctx, &logger, &req, pvc, pv)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+
 	rsCronJob, err := r.findChildCronJob(ctx, &logger, &req)
```

Comment:
move the reclaimspacecron job to its own helper function

Reply:
You mean the entirety of reclaimspace operations? Like the way it is done for key rotation?

Reply:
yes correct